Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -51,18 +51,26 @@ - (void)createSimulatorWithDeviceName:(NSString *)deviceName completion:(void (^
         [BPUtils printInfo:ERROR withString:@"SimDeviceSet failed: %@", [error localizedDescription]];
         return;
     }
+    if (self.config.testing_BluepillCliTest == TRUE) {
```

Comment:
Don't compare to `TRUE`, just `if (self.config.testing_BluepillCliTest)`. 

However, it seems the clone simulator logic isn't being tested right? It's being disabled by the tests.

Reply:
This flag is to make sure Bluepill-cli can be tested alone, that flag is to let Bluepill-cli create simulator instead of clone simulator.